src/Harald.Infrastructure.Slack/SlackFacade.cs

```diff
@@ -26,17 +27,23 @@ public class SlackFacade : ISlackFacade
         private readonly HttpClient _client;
         private readonly IDistributedCache _cache;
         private readonly SlackOptions _options;
+        private readonly string _botUserId;
 
         public SlackFacade( HttpClient client = null, IOptions<SlackOptions> options = null, IDistributedCache cache = null)
         {
             _client = client ?? new HttpClient() { BaseAddress = new System.Uri("https://slack.com", System.UriKind.Absolute) };
             _cache = cache;
             _options = options?.Value;
+            _botUserId = Environment.GetEnvironmentVariable("SLACK_API_BOT_USER_ID") ?? "";
         }
 
+        public string GetBotUserId()
```

Comment:
Consider using an expression bodied member (https://davefancher.com/2014/08/25/c-6-0-expression-bodied-members/) and reserve public methods for actual operations on the subsystem (Slack API). This makes it more intuitive to work with the class which is already bloated enough.

Reply:
Check e9c5937 and 1094b00